### ISSUE TITLE
Fail to generate code

### ISSUE THREAD

**Issue by max2liu** [sngular/scs-multiapi-plugin#142]
yml:
```yaml
asyncapi: '2.5.0'
info:
  title: Streetlights API
  version: '1.0.0'
  description: |
    The Smartylighting Streetlights API allows you
    to remotely manage the city lights.
  license:
    name: Apache 2.0
    url: 'https://www.apache.org/licenses/LICENSE-2.0'
servers:
  mosquitto:
    url: solace://test.solace.org
    protocol: solace
defaultContentType: application/json
channels:
  'person/{personId}/{eventType}':
    subscribe:
      operationId: sendPersonEvent
      description: 'send Person create or update Event'
      bindings:
        solace:
          bindingVersion: 0.3.0
          destinations:
            - destinationType: topic
              topic:
                topicSubscriptions:
                  - person/>
      message:
        $ref: '#/components/messages/PersonMsg'
    parameters:
      personId:
        schema:
          type: string
      eventType:
        schema:
          type: string
  'person/*':
    publish:
      operationId: receivedPersonEvent
      description: 'receive Person create or update Event'
      bindings:
        solace:
          bindingVersion: 0.3.0
          destinations:
            - destinationType: queue
              queue:
                name: CreatedHREvents
                topicSubscriptions:
                  - person/*/created
            - destinationType: queue
              queue:
                name: UpdatedHREvents
                topicSubscriptions:
                  - person/*/updated
      message:
        $ref: '#/components/messages/PersonMsg'
components:
  messages:
    PersonMsg:
      contentType: application/json
      headers:
        type: object
        properties:
          carrier_id:
            type: string
            minimum: 4
            example: 'A'
      payload:
        $ref: '#/components/schemas/PersonEvent'
  schemas:
    PersonEvent:
      type: object
      properties:
        id:
          type: integer
          minimum: 0
          description: Id of person.
        name:
          type: string
          minimum: 6
          description: Name of person.
        sentAt:
          type: string
          format: date-time
          description: Date and time when the message was sent.
```
pom.xml:

<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>

    <parent>
        <groupId>com.oocllogistics.pod</groupId>
        <artifactId>pod_parent</artifactId>
        <version>0.0.1-SNAPSHOT</version>
        <relativePath/>
    </parent>
    <artifactId>demo_project</artifactId>
    <version>0.0.1-SNAPSHOT</version>
    <name>demo_project</name>
    <description>demo project</description>
    <dependencies>

    </dependencies>
    <build>
        <directory>${project.basedir}/target</directory>
        <plugins>
            <plugin>
                <groupId>net.coru</groupId>
                <artifactId>scs-multiapi-maven-plugin</artifactId>
                <version>3.2.0</version>
                <executions>
                    <execution>
                        <phase>generate-sources</phase>
                        <goals>
                            <goal>asyncapi-generation</goal>
                        </goals>
                        <configuration>
                            <fileSpecs>
                                <specFile>
                                    <filePath>${project.basedir}/asyncapi/asyncAPI.yml</filePath>
                                    <consumer>
                                        <ids>publishOperation</ids>
                                        <classNamePostfix>MY_CONSUMER_CLASS</classNamePostfix>
                                        <modelNameSuffix>DTO</modelNameSuffix>
                                        <apiPackage>net.coru.apigenerator.asyncapi.business_model.model.event.consumer</apiPackage>
                                        <modelPackage>net.coru.apigenerator.asyncapi.business_model.model.event</modelPackage>
                                    </consumer>
                                    <supplier>
                                        <ids>subscribeOperation</ids>
                                        <apiPackage>net.coru.apigenerator.asyncapi.business_model.model.event.producer</apiPackage>
                                        <modelPackage>net.coru.apigenerator.asyncapi.business_model.model.event</modelPackage>
                                    </supplier>
                                    <streamBridge>
                                        <ids>streamBridgeOperation</ids>
                                        <apiPackage>net.coru.apigenerator.asyncapi.business_model.model.event.producer</apiPackage>
                                        <modelPackage>net.coru.apigenerator.asyncapi.business_model.model.event</modelPackage>
                                    </streamBridge>
                                </specFile>
                            </fileSpecs>
                            <generatedSourcesFolder>generated-sources</generatedSourcesFolder>
                        </configuration>
                    </execution>
                </executions>
            </plugin>
        </plugins>
    </build>

</project>

After mvn install, I found no code generated under related package in folder generate-sources/apigenerator

**Comment by dianaGomezGar** [comment]
Hi @max2liu, 

There is a mistake in your configuration. The IDs you defined in the plugin config have to be the same that you named in your fileSpec (the 'operationId' field). 

You can see a full example with explanation in the readme.md file.

Can you try to use the same IDs and tell me if that works?

**Comment by max2liu** [comment]
Hi @dianaGomezGar 

Thanks for your help, that is helpful. And there is another question that, if there is any template function for me to customize the code? eg: my project use lombok, how can I customize the model class to use lombok?

**Comment by dianaGomezGar** [comment]
Hi, 

There is a section in the readme file that includes a basic configuration to use this plugin, a few initial considerations and an usage section where you can find a table with includes all specFiles tags you can use to customize your generated classes. Here is the tag you need for use Lombok annotation and more. 

https://github.com/corunet/scs-multiapi-plugin#readme

I hope that helps.

**Comment by max2liu** [comment]
Hi @dianaGomezGar 

Which scs-multiapi-maven-plugin version you mean for useLombokModelAnnotation, I used 3.2.0, met this exception:

> _Cannot find 'useLombokModelAnnotation' in class net.coru.api.generator.plugin.asyncapi.parameter.FileSpec_

this is my code:

```xml
<plugin>
                <groupId>net.coru</groupId>
                <artifactId>scs-multiapi-maven-plugin</artifactId>
                <version>3.2.0</version>
                <executions>
                    <execution>
                        <phase>generate-sources</phase>
                        <goals>
                            <goal>asyncapi-generation</goal>
                        </goals>
                        <configuration>
                            <fileSpecs>
                                <specFile>
                                    <filePath>${project.basedir}/asyncapi/asyncAPI.yml</filePath>
                                    <consumer>
                                        <ids>receivedPersonEvent</ids>
                                        <classNamePostfix>MY_CONSUMER_CLASS</classNamePostfix>
                                        <modelNameSuffix>DTO</modelNameSuffix>
                                        <apiPackage>net.coru.apigenerator.asyncapi.business_model.model.event.consumer</apiPackage>
                                        <modelPackage>net.coru.apigenerator.asyncapi.business_model.model.event</modelPackage>
                                    </consumer>
                                    <streamBridge>
                                        <ids>sendPersonEvent</ids>
                                        <apiPackage>net.coru.apigenerator.asyncapi.business_model.model.event.producer</apiPackage>
                                        <modelPackage>net.coru.apigenerator.asyncapi.business_model.model.event</modelPackage>
                                    </streamBridge>
                                    <useLombokModelAnnotation>true</useLombokModelAnnotation>
                                </specFile>
                            </fileSpecs>
                            <generatedSourcesFolder>generated-sources</generatedSourcesFolder>
                        </configuration>
                    </execution>
                </executions>
            </plugin>
```

**Comment by domoran** [comment]
I also struggled on the versions, the first example on the github page mentions 3.2.0 for maven integration which I blindly copy pasted and ran into the same troubles. What should be done is to have the documentation of the project be generated via Github Pages and fill in the latest release version to the examples automatically. I will open an issue for this

**Comment by dianaGomezGar** [comment]
Hi @max2liu, @domoran, 

We just saw that our examples are a bit outdated, so we are going to create a Wiki page here where we will keep updated all the documentation for this project. 

We will let you know something as soon as possible, please be patience until then. 

Thank you so much.

**Comment by jemacineiras** [comment]
Hi @max2liu, @domoran,

looks like this issue has been addressed in #152, so I'm closing this one.

If you still have the issue, please reopen this in orden to help you.

Cheers
